inspirehep/modules/theme/static/js/plots.js

```diff
@@ -34,11 +34,13 @@
       var id = parseInt(id);
       $('#plotsCarousel').carousel(id);
     });
-    
+
     // When the carousel slides, auto update the text
-    $('#plotsCarousel').on('slid.bs.carousel', function (e) {
+    $('#plotsCarousel').on('slide.bs.carousel', function (e) {
+      setTimeout(function(){
       var id = $('.item.active').data('slide-number');
       $('#carousel-text').html($('#slide-content-'+id).html());
+    },1000)
```

Comment:
Given that JavaScript has function scope, it's key to use indentation to help other programmers understand the scoping of the variable you use. Basically, if you are inside of a function, you indent one level deeper than the level at which the function is defined.